examples/imitation_learning_openbot/policies.py

```diff
@@ -5,17 +5,17 @@
 import numpy as np
 import tflite_runtime.interpreter as tflite
 
+import spear
 from utils import get_compass_observation, get_relative_target_pose
 
-
 class OpenBotPIDPolicy():
```

Comment:
`OpenBotWaypointFollowingPolicy` because it isn't really implementing a PID controller.

Reply:
Updated this.